### PR TITLE
Basic redux wireup

### PR DESCRIPTION
**Pull request by tylerlan**

### CODE REVIEW

File: src/utils/Api.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 export default class Api {
-  static fetchJourneys(origin, destination, time) {
+  static fetchJourneys(origin, destination) {
     return fetch(
-      `http://localhost:8000/directions?origin=${origin}&destination=${destination}&time=${time}`,
-    ).then(response => response.json());
+      `http://localhost:8000/directions?origin=${origin}&destination=${destination}&alternatives=true`,
+    )
+      .then(response => response.json())
+      .catch(e => e);
```
Comment (HamidAghdaee) [inline src/utils/Api.js:7]: What's going on here? you should probably do something meaningful here.

**Comment by HamidAghdaee** [comment]
@tylerlan, tylerlan:basic-redux-wireup is your latest code to be merged here, right? 

I checked it out and ran it. The tests fail
screen shot 2017-08-04 at 4 22 48 pm

Also, the page seems to crash the moment I add a destination. 
screen shot 2017-08-04 at 4 23 29 pm

**Comment by tylerlan** [comment]
@HamidAghdaee 
1) Those tests for timer are not working right now. 
2) That rejection error happens when the api is unable to parse your input. I would have to observe your behavior adding a destination to know what went wrong here. Most of the time it works without being rejected.

**Comment by tylerlan** [comment]
ALSO, if you tried to add 44 Tehama as a Destination, it crashes because that location is hardcoded as the origin.

**Comment by HamidAghdaee** [comment]
@tylerlan Please address both of these issues and I'll merge this PR in. The tests failing and the 44 tehama street situation. thanks

**Comment by HamidAghdaee** [comment]
@tylerlan Let me know when you guys fix these issues so I can merge this.

thanks.

**Comment by tylerlan** [comment]
@HamidAghdaee the issues you raised should be handled now

**Pull request merged**